src/main/java/org/openrewrite/java/migrate/apache/commons/io/ApacheCommonsFileUtils.java

```diff
@@ -61,5 +66,70 @@ void after(File a, String s) throws Exception {
         }
     }
 
+    // file, string, boolean
+    private static class WriteStringToFileOverloadOne {
+        @BeforeTemplate
+        void before(File a, String data, boolean append) throws Exception {
+            FileUtils.writeStringToFile(a, data, append);
+        }
+
+        @AfterTemplate
+        void after(File a, String data, boolean append) throws Exception {
+            Files.write(a.toPath(), data.getBytes(), append ? StandardOpenOption.APPEND : null);
```

Comment:
It looks like `data` is nullable, and therefore we can't immediately call `.getBytes()` here or above. :/ Guess that was missed in the earlier review. Not sure if we should then continue with these recipes, as a ternary for `data` substantially hurts the readability.

Reply:
It looks like `Files.writeString` would have been a better candidate; that said given the nullability issues I doubt if we should continue with these recipes. 🤔